zenodb_test.go

```diff
@@ -463,6 +465,27 @@ ORDER BY _time`
 	})
 }
 
+func testPercentileOptimizedQuery(wg *sync.WaitGroup, t *testing.T, db *DB, includeMemStore bool, epoch time.Time, resolution time.Duration) {
+	defer wg.Done()
+
+	sqlString := `
+SELECT PERCENTILE(pp_5p, 90) AS pp_opt
```

Comment:
This shows an example of what's now possible.

Reply:
Cool!  Thanks also, the context is helpful.